Task 30

eisina opened the pull request.

eisina committed:
Initial

eisina committed:
Test

eisina committed:
Test

eisina committed:
Fixing

eisina committed:
Fixing comments

KBugrimCS commented on pom.xml:
```diff
@@ -0,0 +1,42 @@
+    </properties>
+
+
+    <dependencies>
+    <dependency>
+        <groupId>org.testng</groupId>
+        <artifactId>testng</artifactId>
+        <version>${testng.version}</version>
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.seleniumhq.selenium</groupId>
+        <artifactId>selenium-java</artifactId>
+        <version>${selenium.version}</version>
+    </dependency>
+</dependencies>
```
Align the file according to the required tabs

KBugrimCS commented on src/test/coherentsolutions.web.test/java/Managers/WebDriverManager.java:
```diff
@@ -0,0 +1,15 @@
+
+public class WebDriverManager {
+
+    static WebDriver driver;
```
do not forget about access modifier

KBugrimCS commented on src/test/coherentsolutions.web.test/java/YandexTest.java:
```diff
@@ -0,0 +1,48 @@
+
+    private WebDriver driver;
+
+    private final String PASSWORD = "testik1234";
+    private final String LOGIN = "katetestng";
```
Please follow java code convention in the fields naming

KBugrimCS commented on src/test/coherentsolutions.web.test/java/YandexTest.java:
```diff
@@ -0,0 +1,48 @@
+        driver = WebDriverManager.initDriver();
+    }
+
+    @Test
```
It would be nice if you add description parameter to the annotation which describe what you do in this test

KBugrimCS commented on README.md:
```diff
@@ -0,0 +1 @@
+# Testik
```
Please add valid information about your project to the Readme file

KBugrimCS reviewed (commented):
Please update your pr according to the comments. Also add useless files to gitignore

eisina committed:
Adding gitignore

eisina committed:
Remove .idea files from commit

eisina committed:
ReadMe modified
Alignment modified in all files
Adding access modifier to Driver Class
Adding logging
Moving credentials to Property file and adding class to handle it
Add test description

eisina committed:
Remove .gitignore idea file from commit

eisina committed:
Adding quit method
